.github/workflows/e2e.yml

```diff
@@ -33,3 +33,16 @@ jobs:
         run: docker run --rm -d --name subscription-example-image -p 8080:8080 -e ADYEN_API_KEY="${{ secrets.ADYEN_API_KEY }}" -e ADYEN_MERCHANT_ACCOUNT=${{ secrets.ADYEN_MERCHANT_ACCOUNT }} -e ADYEN_CLIENT_KEY=${{ secrets.ADYEN_CLIENT_KEY }} -e ADYEN_HMAC_KEY=${{ secrets.ADYEN_HMAC_KEY }} subscription-example-image:latest
       - name: Run testing suite against subscription-example
         run: docker run --rm --name adyen-testing-suite -e PLAYWRIGHT_FOLDERNAME=subscription --network host ghcr.io/adyen-examples/adyen-testing-suite:main
+
+giftcard:
```

Comment:
Giftcard E2E fails to run due to invalid .yml spacing, see: https://github.com/adyen-examples/adyen-node-online-payments/actions/runs/4923902883